lib/codecs/src/decoding/mod.rs

```diff
@@ -236,7 +236,20 @@ pub enum DeserializerConfig {
 
     /// Decodes the raw bytes as a [GELF][gelf] message.
     ///
+    /// This codec is experimental for the following reason:
+    ///
+    /// The GELF specification is more strict than the actual Graylog receiver.
+    /// Vector's decoder currently adheres more strictly to the GELF spec, with
+    /// the exception that some characters such as `@`  are allowed in field names.
+    ///
+    /// Other GELF codecs such as Loki's, use an [implementation][implementation] that is maintained
+    /// by Graylog, and is much more relaxed than the GELF spec.
+    ///
+    /// Going forward, Vector will use that as the reference implementation, which means
```

Comment:
```suggestion
    /// Going forward, Vector will use that as the [Go SDK][implementation] as the reference implementation, which means
```

Reply:
🤔  did you want to have [Go SDK] on line 245 as well? So:

`Other GELF codecs such as Loki's, use a [Go SDK][implementation] that is maintained` ?

And adjusting the grammar a little on line 248:

`Going forward, Vector will use that [Go SDK][implementation] as the reference implementation, which means`

Reply:
👍 that makes sense.